composer.json

```diff
@@ -22,14 +22,14 @@
         "sake"
     ],
     "require": {
-        "bramus/monolog-colored-line-formatter": "~2.0",
-        "composer/installers": "~1.0",
-        "embed/embed": "^3.0",
+        "bramus/monolog-colored-line-formatter": "^2",
```

Comment:
`~2.0` is equivalent in every way to `^2`.

`~2.0` allows the installation of `~2.1` ... `~2.0.0` does not.

https://getcomposer.org/doc/articles/versions.md#tilde-version-range-